modules/home-manager/fcitx5.nix

```
@@ -5,7 +5,12 @@ let
   inherit (config.catppuccin) sources;
 
   cfg = config.catppuccin.fcitx5;
-  enable = cfg.enable && config.i18n.inputMethod.enabled == "fcitx5";
+  enable =
+    cfg.enable
+    && (
+      (config.i18n.inputMethod ? enable && config.i18n.inputMethod.type == "fcitx5")
```

This isn't actually checking if `config.i18n.inputMethod.enable` is true.

e.g.

```console
nix-repl> foo = { bar = false; }
Added foo.

nix-repl> foo ? bar
true

nix-repl> foo ? bar && foo.bar
false
```

```suggestion
      (config.i18n.inputMethod ? enable && config.i18n.inputMethod.enable && config.i18n.inputMethod.type == "fcitx5")
```